saylua/modules/messages/models/db.py

```diff
@@ -1,6 +1,45 @@
 from google.appengine.ext import ndb
 
 import datetime
+from saylua import db
+
+
+class Conversation(db.Model):
+    __tablename__ = "conversations"
+
+    id = db.Column(db.Integer, primary_key=True)
+    title = db.Column(db.String(256))
+    author = db.Column(db.Integer, db.ForeignKey('users.id'))
+
+
+class ConversationMember(db.Model):
+    __tablename__ = "conversation_members"
+
+    conversation_id = db.Column(db.Integer, db.ForeignKey('conversations.id'), primary_key=True)
+    user_id = db.Column(db.Integer, db.ForeignKey('users.id'), primary_key=True)
+    unread = db.Column(db.Boolean, default=False)
+
```

Comment:
Maybe we could add like a .name() function to the user model that was a wrapper around this? Agree with changing display_name.display_name regardless haha.